app/src/main/res/layout/widget_button.xml

```diff
@@ -10,44 +10,51 @@
     android:background="@drawable/widget_button_background"
     android:padding="4dp">
 
-    <LinearLayout
+    <RelativeLayout
         android:id="@+id/widgetImageButtonLayout"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:paddingVertical="4dp"
+        android:layout_height="wrap_content"
+        android:paddingVertical="8dp"
+        android:paddingHorizontal="4dp"
         android:orientation="vertical">
 
         <ImageView
+            android:id="@+id/widgetImageButtonBackground"
+            android:layout_width="40dp"
+            android:layout_height="40dp"
+            android:translationZ="-2dp"
+            android:alpha="0.4"
+            android:contentDescription="@string/widget_button_image_description"
```

Comment:
We don't need the same accessibility descriptions on both the background and the icon. I think you can just leave it out here / mark it as no description needed.